### ISSUE TITLE
swCSTp docs should be clearer on what output is

### ISSUE THREAD

**Issue by richardsc** [dankelley/oce#863]
For example the docs for `swCSTp()` say:

> Compute electrical conductivity based on salinity, temperature, and pressure.

and it's not until the "Value" section that it's make explicit that it outputs conductivity *ratio*. This is true for `eos='gsw'` even though the underlying `gsw` function actually outputs mS/cm.

I propose updating the docs to be clearer about conductivity ratio vs conductivity. Another idea would be to rename the function (or create an alias) that makes it clearer from the function name what the return value is, e.g. `swCRSTp()`, but I don't think that's actually any better.

**Comment by dankelley** [comment]
Are you altering it or asking me to? We don't want to get into crossed edits.  (PS. right now I'm trying to redo the gsw vignette in markdown so it can pass travis build tests so you won't conflict if you edit and push.)  It's just one word.

**Comment by richardsc** [comment]
I assigned myself. I'll do it.

**Comment by richardsc** [comment]
Fixed in commit 63febc0a49aa3. I'd appreciate a second opinion, since I managed to confuse myself about what the "reference" conductivity is in relation to T90 vs T68, and what one should expect if they do:
```r
swCSTp(35, 15, 0)
```
(i.e. **not** `1`).

**Comment by dankelley** [comment]
Thanks, I'll look. Also,  think we need a citation for this number `42.9140` that we state. It should be very specific, narrowed to an eqn if there is one.

I'm having real troubles getting the gsw vignette to work and will likely focus on that tonight. I think Hadley has a typo in some of his code (although I don't think that's the problem I'm getting).

**Comment by richardsc** [comment]
After some asking and digging, I think this is the reference:

@article{culkin1980determination,
  title={Determination of the concentration of potassium chloride solution having the same electrical conductivity, at 15 C and infinite frequency, as standard seawater of salinity 35.0000‰(Chlorinity 19.37394‰)},
  author={Culkin, F and Smith, Norman D},
  journal={Oceanic Engineering, IEEE Journal of},
  volume={5},
  number={1},
  pages={22--23},
  year={1980},
  publisher={IEEE}
}

I can't find a pdf but I have a printed copy. I'll update more after I've read it a bit.

**Comment by richardsc** [comment]
Here's the data table from that paper that the value comes from:
```
| Batch No. | Chlorinity (ppt) | Conductivity (mmho/cm at 15 degC |
|-----------+------------------+----------------------------------|
| P59       |           19.377 |                          42.9202 |
| P60       |          19.3765 |                          42.9193 |
| P68       |          19.3718 |                          42.9095 |
| P71       |          19.3740 |                          42.9139 |
| P72       |          19.3748 |                          42.9163 |
| P75       |          19.3740 |                          42.9146 |
| P77       |          19.3755 |                          42.9168 |
| P79       |          19.3760 |                          42.9178 |
| P80       |          19.3782 |                          42.9219 |
| P82       |          19.3706 |                          42.9075 |
| P84       |          19.3762 |                          42.9181 |
```

The authors state that a linear regression of conductivity on chlorinity gives:

        C = 5.325053 + 1.940181 * Cl(ppt)

and the value at 19.37394 is 42.914 mS/cm. I thought I'd check it, and put some uncertainty on it. See below:
![cond_to_chl](https://cloud.githubusercontent.com/assets/233584/12863894/76c62ea4-cc4b-11e5-84fc-e4164ebe38c7.png)

That plot is way too complicated, but here are the high points:

1. A linear regression prediction of the conductivity at a chlorinity of 19.37394 gives 42.914005 +- 0.000286 mS/cm. This is encompasses the stated value of 42.914 *however* I get slightly different coefficients from them (see upper left)

2. Just using their formula gives C35150=42.91404, again, basically 42.914 given the uncertainty.

One thing I don't understand (yet) is that the chlorinity value they report is for S=35 ppt, but I think 35 ppt is not the same as 35.000 on the PSS. Probably still not enough to make a difference.

That was kind of fun. I'll add the reference just for the fun of it.

**Comment by richardsc** [comment]
Added reference in commit 1649de9ead819a211. Feel free to close if you're happy with it.

**Comment by dankelley** [comment]
Good detective work!!  I see this value 42.914 in 12 places in the code and docs. I suppose each instance in the docs should cite this Culkin and Smith paper, so I'll do that and close the issue when I'm done.

Also, I added a line in the doc for `swCSTp`:
```r
swCSTp(35, T90fromT68(15), 0, eos="unesco") # 1, by definition of cond. ratio
```
which I think may help users a bit.

NOTE: the line I added to the doc is already in the test suite.
